src/stop.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Why not `/bin/bash`?

Reply:
/usr/bin/env bash is to find bash in environment variables, in most cases, it is the same. /usr/bin/env bash is more flexibility.

Reply:
also, on FreeBSD, bash is not necessarily installed into /usr/bin/, normally, it is `/usr/local/bin/bash`.